recipes/mcmc-sampling-emcee/mcmc_sampling.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        nwalkers, ndim, lnprob, args=[dataset], #pool=pool
</pre><p><code>lnprob</code> trakes two arguments. What are they?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
the list of parameters (the free params in the model) and the gammapy dataset.
I've added a description in the lnprob() function definitoin (see previsou comment).